src/specs/specs-other.json

```diff
@@ -1,8 +1,24 @@
 [
   "https://streams.spec.whatwg.org/",
+  "https://w3c.github.io/accname/",
+  "https://w3c.github.io/core-aam/",
   "https://w3c.github.io/editing/contentEditable.html",
+  "https://w3c.github.io/editing/execCommand.html",
+  "https://webassembly.github.io/spec/",
```

Comment:
@tidoust I see you've excluded TC39 specs. Do you want to exclude this too?

Reply:
This was because I hadn't updated the manual fixups for day-to-day. I've done so in https://github.com/foolip/day-to-day/commit/f01e6dc78faa8ba0ea64c87a9c44ebf1d5994625 and will revert this change.

Crawling https://webassembly.github.io/spec/core/ might be nice, but it's multi-page and would probably need some special treatment.

Reply:
I don't know if all WebAssembly variants are kept in sync automatically, but adding https://www.w3.org/TR/wasm-core-1/ would mean that the corresponding Editor's Draft at https://webassembly.github.io/spec/core/bikeshed/ gets crawled, and that one is single page.